plugins/action_generate_text_string/package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "generate_name",
+  "name": "generate_text_string",
   "version": "0.1",
-  "description": "Creates a text string and (optionally) applies it to a record, e.g. an application name",
+  "description": "Generates serial numbers or arbitrary text strings using a specified pattern",
```

Comment:
Should it keep the `e.g. an application name` as this is the default location? Or it isn't ?

Reply:
Good call: b149be66